### DISCUSSION TITLE
Custom themes with changing colors?

### DISCUSSION THREAD

**Discussion by akhilesh-balaji** [monkeytypegame/monkeytype#2350] [Ideas] [answered]
The themes `fire` and `rgb` have this really cool effect where the main accent colour keeps changing. It would be great if there was a way to create themes like this from the custom theme creator. Is there already a way to do this?

**Comment by unrealapex** [comment, marked as answer]
I think creating a user interface for creating animations would be far too complex for the scope of this project. Having an option to use custom CSS(link to stylesheet or pasting properties directly) for the background would be easier and more customizable. What do you think @Miodec?

**Reply by Miodec** [reply]
Yeah, custom themes only support just changing the hex colors. If you want to do truly custom themes, i think best way is to just install a styling extension like Stylus.
